crates/autopilot/src/arguments.rs

```diff
@@ -363,6 +363,10 @@ pub struct FeePolicy {
     /// filled.
     #[clap(long, env, action = clap::ArgAction::Set, default_value = "true")]
     pub fee_policy_skip_market_orders: bool,
+
+    /// The number of fee policies to insert in a single batch.
+    #[clap(long, env, default_value = "500")]
+    pub fee_policies_insert_batch_size: NonZeroUsize,
```

Comment:
Do we foresee any reason for why this value should be different from `order_events_insert_batch_size`? If so, can we combine them in a database related struct (which can be flattened on the top args, but forms a place to but db related configs, maybe also with other configs) or rename `order_events_insert_batch_size` and reuse it?

Reply:
Different values could be theoretically required since data size differs for a single row in different tables. For now,  we can probably combine them into a single config param to simplify things.